apps/wing/src/commands/compile.ts

```diff
@@ -227,3 +280,13 @@ function resolvePluginPaths(plugins: string[]): string[] {
   }
   return resolvedPluginPaths;
 }
+
+function offsetFromLineAndColumn(source: string, line: number, column: number) {
```

Comment:
Too bad I recently removed the byte offsets from Wingspan because we weren't using them, that probably would've been a good optimization here.

Reply:
Why not add them back?

Reply:
#1815